clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/ApplicationContextAtomicOperationsRegistry.groovy

```diff
@@ -31,12 +32,22 @@ class ApplicationContextAtomicOperationsRegistry implements AtomicOperationsRegi
   ApplicationContext applicationContext
 
   @Override
-  AtomicOperationConverter getAtomicOperationConverter(String description, String cloudProvider) {
-    (AtomicOperationConverter) applicationContext.getBean(description)
+  AtomicOperationConverter getAtomicOperationConverter(String description, String cloudProvider, ProviderVersion version) {
+    def result = (AtomicOperationConverter) applicationContext.getBean(description)
+    if (!result.acceptsVersion(version)) {
```

Comment:
I think they can still implement `acceptsVersion()`, it was just to reduce boilerplate code at the call-sites. The exception messages and everything else match.